clients/src/test/java/org/apache/kafka/common/config/ConfigDefTest.java

```diff
@@ -100,6 +100,21 @@ public void testMissingRequired() {
         new ConfigDef().define("a", Type.INT, Importance.HIGH, "docs").parse(new HashMap<String, Object>());
     }
 
+    @Test
+    public void testRangeToString() {
```

Comment:
Intentionally did not test all variations of the `toString()` as I'm not sure we expect users to explicitly pass in a `null` `min` field to one of the public constructor methods